src/core/common/utility.go

```diff
@@ -1326,6 +1328,45 @@ func GetAvailableK8sVersion(providerName string, regionName string) (*[]model.K8
 	providerName = strings.ToLower(providerName)
 	regionName = strings.ToLower(regionName)
 
+	// Debug: Check file system directly
+	possiblePaths := []string{
+		"/app/assets/k8sclusterinfo.yaml",
+		"./assets/k8sclusterinfo.yaml",
+		"../assets/k8sclusterinfo.yaml",
+		"assets/k8sclusterinfo.yaml",
+	}
+
+	for _, path := range possiblePaths {
+		if fileInfo, err := os.Stat(path); err == nil {
+			log.Debug().
+				Str("filePath", path).
+				Time("modTime", fileInfo.ModTime()).
+				Int64("fileSize", fileInfo.Size()).
+				Msg("Found k8sclusterinfo.yaml")
+
+			// Search for ncpvpc in file content
+			if content, err := os.ReadFile(path); err == nil {
+				hasNcpvpcInFile := strings.Contains(string(content), "ncpvpc:")
+				log.Debug().
+					Str("filePath", path).
+					Bool("hasNcpvpcInFile", hasNcpvpcInFile).
+					Int("contentLength", len(content)).
+					Msg("File content analysis")
+			}
+			break
+		}
+	}
+
+	// Debug: Check file path and content
+	log.Debug().Interface("RuntimeK8sClusterInfo", RuntimeK8sClusterInfo).Msg("Current RuntimeK8sClusterInfo content")
```

Comment:
Sorry, but this section will not be accepted.

File paths for the configuration should be centrally handled around [main.go#L241](https://github.com/cloud-barista/cb-tumblebug/blob/main/src/main.go#L241)
, not within individual Go modules.

Also, this debug output seems to be for your own testing rather than for everyone. :) You can run it locally if you wish, but it doesn’t seem appropriate for upstream.

If you think this debugging code is important for upstream, see the [main.go#L241](https://github.com/cloud-barista/cb-tumblebug/blob/main/src/main.go#L241) code and look for more appropriate place.